src/comparators.js

```diff
@@ -19,8 +38,23 @@ export function compareTracks(rootState, t1, t2) {
     return -1;
   }
 
-  let albOrd = compareStrings(a1.normalized_title, a2.normalized_title);
-  albOrd = albOrd === 0 ? compareStrings(a1.release, a2.release) : albOrd;
-  albOrd = albOrd === 0 ? compareStrings(a1.id, a2.id) : albOrd;
+  let albOrd = compareAlbumsByTitle(a1.normalized_title, a2.normalized_title);
   return albOrd === 0 ? t1.number - t2.number : albOrd;
 }
+
+export function compareAlbumsByTitle(a1, a2) {
```

Comment:
These function names aren't accurate, IMO. It would be clearer to say `compareAlbumsByTitleFirst` or `compareForAlbumsSortedByTitle` (or something else that conveys that the albums are compared by more than their titles).